operators/hpe-csi-operator/deploy/olm-catalog/hpe-csi-driver-operator/1.4.0/hpe-csi-operator.v1.4.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,335 @@
+              resources: ["csinodes"]
+              verbs: ["get", "list", "watch"]
+          # Need the same permissions as external-snapshotter clusterrole to be able to create it
+            - apiGroups: ["snapshot.storage.k8s.io"]
+              resources: ["volumesnapshots"]
+              verbs: ["create", "update", "delete", "get", "list", "watch"]
+            - apiGroups: ["snapshot.storage.k8s.io"]
+              resources: ["volumesnapshots/status"]
+              verbs: ["update"]
+            - apiGroups: ["snapshot.storage.k8s.io"]
+              resources: ["volumesnapshotcontents"]
+              verbs: ["create", "update", "delete", "get", "list", "watch"]
+            - apiGroups: ["snapshot.storage.k8s.io"]
+              resources: ["volumesnapshotcontents/status"]
+              verbs: ["create", "get", "list", "watch", "update", "delete"]
+            - apiGroups: ["snapshot.storage.k8s.io"]
+              resources: ["volumesnapshotclasses"]
+              verbs: ["get", "list", "watch"]
+          # Need the same permissions as driver-registrar clusterrole to be able to create it.
+            - apiGroups: ["apiextensions.k8s.io"]
+              resources: ["customresourcedefinitions"]
+              verbs: ["get", "watch", "list", "delete", "update", "create"]
+            - apiGroups: ["csi.storage.k8s.io"]
+              resources: ["csidrivers"]
+              verbs: ["get", "watch", "list", "delete", "update", "create", "patch"]
+            - apiGroups: ["storage.k8s.io"]
+              resources: ["csidrivers"]
+              verbs: ["get", "watch", "list", "delete", "update", "create", "patch"]
+          # Need the same permissions as external-resizer to be able to create it.
+            - apiGroups: [""]
+              resources: ["persistentvolumeclaims/status"]
+              verbs: ["create", "get", "list", "watch", "update", "delete", "patch"]
+          # Need permissions to let csi-driver load/unload/update hpenodeinfos resources
+            - apiGroups: ["storage.hpe.com"]
+              resources: ["hpenodeinfos"]
+              verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+          # Need permissions to let 3PAR CSP to load/unload/update hpevolumeinfos resources
+            - apiGroups: ["storage.hpe.com"]
+              resources: ["hpevolumeinfos"]
+              verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+            - apiGroups: ["storage.hpe.com"]
+              resources: ["hpereplicationdeviceinfos"]
+              verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+            - apiGroups: ["coordination.k8s.io"]
+              resources: ["leases"]
+              verbs: ["get", "watch", "list", "delete", "update", "create", "patch"]
+            - apiGroups: ["batch"]
+              resources: ["jobs"]
+              verbs: ["get", "watch", "list", "delete", "update", "create"]
```

Comment:
Do we need to add 
  - apiGroups: ["storage.hpe.com"]
    resources: ["hpevolumegroupinfos"]
    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
  - apiGroups: ["storage.hpe.com"]
    resources: ["hpesnapshotgroupinfos"]
    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]